client/my-sites/stats/site.jsx

```diff
@@ -275,11 +274,14 @@ class StatsSite extends Component {
 						/>
 
 						{ isSitePrivate ? this.renderPrivateSiteBanner( siteId, slug ) : null }
-						{ ! isSitePrivate && <StatsNoContentBanner siteId={ siteId } siteSlug={ slug } /> }
```

Comment:
I see we're keeping the file `client/my-sites/stats/stats-no-content-banner/index.tsx`. Should we maintain it or do you think it would be good to migrate [the only other usage of it](https://github.com/Automattic/wp-calypso/blob/update/stats-move-writeapost-banner-to-mini-carousel/client/my-sites/stats/stats-email-open-detail/index.jsx#L283) as well?

Reply:
I keeped it because is also used [here](https://github.com/Automattic/wp-calypso/blob/782539d76c393b6c00c36552e1df02b4ac68dd89/client/my-sites/stats/stats-email-open-detail/index.jsx#L283)